<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1105</width>
         <height>669</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="QWidget" name="pageQr">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="1">
             <widget class="QPlainTextEdit" name="plainTextEdit_updateConfig"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_updateConfig">
              <property name="text">
               <string>Update config</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageMenu">
           <layout class="QGridLayout" name="gridLayout"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionColorscale_Report">
   <property name="text">
    <string>Colorscale Report</string>
   </property>
  </action>
  <action name="actionPrepare_config_set">
   <property name="text">
    <string>prepare_config_set</string>
   </property>
  </action>
  <action name="actionSet_calibration_strips">
   <property name="text">
    <string>set calibration strips</string>
   </property>
  </action>
  <action name="actionupdate_config">
   <property name="text">
    <string>update config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
